src/views/EditEvent.tsx

```diff
@@ -11,10 +11,10 @@ import ReturnHome from "../components/ReturnHome";
 const EditEvent = () => {
   // get id from URL
   const path = useLocation().pathname;
-  const id = path.split("/")[2];
+  const id = path.split("/")[2]; // TODO - split to method, add unit test - any better way to do this maybe like useParams?
 
   const localGroups = getLocalGroups();
-  const currentGroup: GroupInfo | undefined = localGroups.find(
+  const currentGroup = localGroups.find(
```

Comment:
If you look at the type return signature for `localGroups.find`, it will automatically give you a type of `GroupInfo | undefined`.  If you get an inferred type from a method, you shouldn't explicitly provide for inline variables since it's redundant.  Explicit types are always helpful on method signatures, but inline variables can often be inferred.